docker/Dockerfile

```diff
@@ -1,7 +1,11 @@
 FROM debian:9.5-slim
 
-# Default google-fluentd at version 1.6.1-1.
-ARG REPO_SUFFIX=20181130-2
+# Default google-fluentd at the unstable version.
+# Use `docker build --build-arg REPO_SUFFIX=<CUSTOM_REPO_SUFFIX>` to build
+# images with a specific version of the package.
+ARG REPO_SUFFIX=unstable
```

Comment:
The key to using an exact repo suffix is that we know exactly what version is getting installed. `unstable` is a movable tag.